smithy-aws-protocol-tests/src/main/resources/META-INF/smithy/ec2-query/input.smithy

```diff
@@ -0,0 +1,362 @@
+// This file defines test cases that test the basics of input serialization.
+
+$version: "0.5.0"
+
+namespace aws.protocols.tests.ec2
+
+use aws.api#ec2QueryName
+use aws.protocols.tests.shared#EpochSeconds
+use aws.protocols.tests.shared#FooEnum
+use smithy.test#httpRequestTests
+
+/// This test serializes strings, numbers, and boolean values.
+operation SimpleInputParams {
+    input: SimpleInputParamsInput
+}
+
+apply SimpleInputParams @httpRequestTests([
+    {
+        id: "Ec2SimpleInputParamsStrings",
+        description: "Serializes strings",
+        protocol: "aws.ec2",
+        method: "POST",
+        uri: "/",
+        headers: {
+            "Content-Type": "application/x-www-form-urlencoded"
+        },
+        body: """
+              Action=SimpleInputParams
+              &Version=2020-01-08
+              &Foo=val1
+              &Bar=val2""",
+        bodyMediaType: "application/x-www-form-urlencoded",
+        params: {
+            Foo: "val1",
+            Bar: "val2"
+        }
+    },
+    {
+        id: "Ec2SimpleInputParamsStringAndBooleanTrue",
+        description: "Serializes booleans that are true",
+        protocol: "aws.ec2",
+        method: "POST",
+        uri: "/",
+        headers: {
+            "Content-Type": "application/x-www-form-urlencoded"
+        },
+        body: """
+              Action=SimpleInputParams
+              &Version=2020-01-08
+              &Foo=val
+              &Baz=true""",
+        bodyMediaType: "application/x-www-form-urlencoded",
+        params: {
+            Foo: "val1",
+            Baz: true,
+        }
+    },
+    {
+        id: "Ec2SimpleInputParamsStringsAndBooleanFalse",
+        description: "Serializes booleans that are false",
+        protocol: "aws.ec2",
+        method: "POST",
+        uri: "/",
+        headers: {
+            "Content-Type": "application/x-www-form-urlencoded"
+        },
+        body: """
+              Action=SimpleInputParams
+              &Version=2020-01-08
+              &Foo=val1
```

Comment:
Looks out of sync with the params (exists here, doesn't in params).

[Also missed in review on the `query` protocol.](https://github.com/awslabs/smithy/pull/250/files#diff-c4a1d71fae24716dec1e6a427fe0dd83R67)